Simplenote.xcodeproj/project.pbxproj

```diff
@@ -4652,6 +4660,14 @@
 /* End XCConfigurationList section */
 
 /* Begin XCRemoteSwiftPackageReference section */
+		3F4BA07C26CDF295000619B1 /* XCRemoteSwiftPackageReference "ScreenObject" */ = {
+			isa = XCRemoteSwiftPackageReference;
+			repositoryURL = "https://github.com/Automattic/ScreenObject";
+			requirement = {
+				branch = "always-compute-at-runtime";
+				kind = branch;
```

Comment:
Once https://github.com/Automattic/ScreenObject/pull/3 is merged and before merging this, I'll ship a new ScreenObject version, and point to it here.